redisearch/client.go

```diff
@@ -148,6 +148,16 @@ type IndexingOptions struct {
 	Replace  bool
 }
 
+// MultiError Represents one or more errors
+type MultiError map[int]error
+
+func (e MultiError) Error() string {
```

Comment:
Wouldn't that require line breaks to make it readable, and wouldn't line breaks confuse a lot of things?